composer.json

```diff
@@ -4,46 +4,58 @@
     "type": "project",
     "description": "Symfony RAD API Edition for powerful REST applications.",
     "autoload": {
-        "psr-4": { "": "src/" }
+        "psr-4": {
+            "AppBundle\\": "src/AppBundle",
+            "DemoBundle\\": "src/DemoBundle"
+        },
+        "classmap": ["app/AppKernel.php", "app/AppCache.php"]
     },
     "require": {
         "php": "^5.6|^7.0",
 
-        "sylius/resource-bundle":               "0.15",
+        "sylius/resource-bundle": "dev-master",
         "incenteev/composer-parameter-handler": "~2.0",
```

Comment:
Let's use `^` instead of `~` in the version constraints to show that dependencies followe semver (here and everywhere else in this file).